tasks/rss-mentor-dashboard.md

```diff
@@ -1,77 +1,83 @@
-# RSS Дашборд ментора
+# RSS Mentor Dashboard
 
 ### Deadline: Sunday, Feb 24, 2019 23:59
 
-## Задание
-Вы должны реализовать дашборд ментора школы The Rolling Scopes в виде странички на github pages.
+## Task
+You need to implement dashboard of The Rolling Scopes School mentor as a page on github pages.
 
-Дашборд должен содержать следующую информацию
-- список студентов, которых курирует ментор
-- список тасков студентов, которые должны делать студенты
-- таблица студент / задания должны быть подсвечена следующим образом:
-    - зеленый цвет - проверенные задания
-    - желтый цвет - задания сейчас в работе у студентов (status - in progress)
-    - красный цвет - задания, которые ментору надо проверять (status - checking)
-    - темно-красный цвет - задания, которые уже надо было сдать, но их не сдали (status - checked)
-    - серая подсветка / притухшие - задания, которые еще не надо делать (status - no status / todo)
+Dashboard should contain following information:
+- list of mentor's students
+- list of student's tasks, which students should work on
+- table student / tasks should be highlighted according to following rules:
+    - green color - reviewed tasks
+    - yellow color - tasks, which students are working on (status - in progress)
+    - red color - tasks, which are awaiting review from mentor (status - checking)
+    - dark red color - tasks, which should already be finished by students,
+    but they were not finished (status - checked)
+    - grey color - tasks, which should not be started yet (status - no status / todo)
 
-Выбор ментора должен происходить из всех менторов набора, которые содержатся в отдельном excel файле, используя контрол автокомплита (https://bestofjs.org/projects/react-select или https://bestofjs.org/projects/chosen). Выбранный ментор должен сохраняться в localStorage и подставляться при следующей загрузке страницы.
+Selection of mentor should be done from all mentors of the current stream,
+which are contained in separated excel file,
+using autocomplete (https://bestofjs.org/projects/react-select or https://bestofjs.org/projects/chosen).
+Selected mentor should be saved in localStorage and used on next page load.
```

Comment:
"Selected mentor should be saved into a localStorage and seen on next page load."

Reply:
fixed